CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+More instructions on [Releasing/Changelog](/docs/RELEASING.md#changelog)
+
+## [0.1.0] - 30/05/2023
+### Added
+
+- Initial release (#4).
+- Add documentation with versioning and changelog instructions (#13).
```

Comment:
GitHub generates a changelog (good enough) when creating a release. I suggest reusing it for now. All releases are available under https://github.com/profusion/android-enhanced-video-player/releases

To generate a new release:

1. On `Code` tab, go to Releases on the right-hand side
2. Click on [Create a new release](https://github.com/profusion/android-enhanced-video-player/releases/new)
3. Create a new version based on the Semantic Versioning
4. Click on `Generate Release Notes`
5. Click on `Publish new Release`

We can only generate releases via this process for now. This would eliminate the need for a `CHANGELOG.md` document.